openmls/src/treesync/node/leaf_node.rs

```diff
@@ -382,6 +384,26 @@ impl TlsDeserializeTrait for LeafNodeTbs {
     }
 }
 
+// TODO: Name up to debate.
+#[derive(Debug, Clone, PartialEq, Eq, TlsSize, TlsDeserialize)]
```

Comment:
Note that this only derives `TlsDeserialize` and not `TlsSerialize` because we don't want so serialize (and send) invalid leaf nodes.